modules/equativBidAdapter.js

```diff
@@ -151,6 +151,17 @@ export const converter = ortbConverter({
       });
     }
 
+    // "assets" is not included as a property to check here because the
+    // ortbConverter library checks for it already and will skip processing
+    // the request if it is missing
+    if (deepAccess(bid, 'mediaTypes.native')) {
+      ['privacy', 'plcmttype', 'eventtrackers'].forEach(prop => {
+        if (!bid.mediaTypes.native.ortb[prop]) {
+          logWarn(`${LOG_PREFIX} Property "${prop}" is missing from request`, bid);
```

Comment:
can you add detail, eg ${LOG_PREFIX} Property "${prop}" is missing from request, ${LOG_PREFIX} will not bid

Reply:
Our product managers want the warnings in there, but do want the bid request to be stopped if these particular properties are missing.  

If `assets` is missing, they are OK with the bid getting killed, but that is handled already by `native.js`.

If not bidding if these other properties are missing is a hard requirement, I can share that back to them.  Or if you would like the message further refined in another way (e.g. adding "it is strongly recommended you add ${prop}", etc.) we can do that too.

Reply:
putting myself in the shoes of someone reading this message, i dont know what the outcome is. Tell the publisher the consequence.